pom.xml

```diff
@@ -341,11 +347,29 @@
           <failOnError>false</failOnError>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
Did you verify the build fails when the above exclusion is removed?

Reply:
Indeed I did. Needed to run to make sure I put the exclusion in the right spot. Confirmed build succeeds when exclusion added.